### DISCUSSION TITLE
How to check the harvesters on the main machine?

### DISCUSSION THREAD

**Discussion by kenneth-1024** [Chia-Network/chia-blockchain#2049] [General]
Hi all,
I followed this wiki page: https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

All my main machine and harvesters are run on linux. I can find the tcp connections between the main machine and the harvesters via "netstat -apnt | grep 8447".

I want to check my harvesters on the main machine, but there is no "Hide Advanced Options" on the "Farm" tab on the linux GUI. And I didn't find any CLI command to do that.

Any suggestions?
Thanks in advanced!

**Comment by miwinski** [comment]
Hi there, 
I have a farmer node running on one Linux box and a separate harvester with its own plots on another Linux machine. I connect remote GUI from MacOS and on the Farm tab i section "Last Attempted proof" i see updates from both harvesters (unfortunately the only way to see that each attempt comes from a different harvester is in a different number of available plots).
Another way to confirm your harvesters are connected to your farmer node is to bump logs to INFO - you'd see things like`new_signage_point_harvester to peer X.X.X.X` where X.X.X.X is an ip of your remote harvester.

**Reply by kenneth-1024** [reply]
Hi there,
Now I can see my harvesters:
![image](https://user-images.githubusercontent.com/7152596/115186410-3c6b1280-a114-11eb-9b3c-d8b940922e30.png)
Both my these two have 50 plots.
But I still can't find the section "Last Attempted proof". 
Is there anything wrong here?

Thanks

**Reply by miwinski** [reply]
![image](https://user-images.githubusercontent.com/1716039/115882182-cedd2000-a44c-11eb-9194-d2a15f0febaa.png)
This is how it looks like for me. I'm using release 1.1.1, gui built from GIT. It looked this same on release 1.0.3. As you can see every other line has a different amount of "Plots Passed Filter" - unfortunately if your harvesters have equal amount of plots there's no way to determine if the attempts are coming from which node the attempt came from. Sounds like a feature request i guess :)

**Reply by salsaly4** [reply]
This is how Farm tab looks in the remote GUI

<img width="1081" alt="Screenshot 2021-04-23 at 17 07 18" src="https://user-images.githubusercontent.com/16805161/115883603-a4906000-a456-11eb-8d0b-80fda7151189.png">

But in the logs I get the harvester connectoins:

`2021-04-23T16:52:52.548 farmer farmer_server              : INFO     -> new_signage_point_harvester to peer 8.215.48.71 3b408fbc1eeec65e5a3304e969513cc3e8ef4bcf05682a1ced197d82d422e03b`

And nothing in the `chia farm summary` on the farmer node.

What I'm doing wrong?

**Comment by salsaly4** [comment]
I have the same problem — I have several harvesters connected to remote full node on Linux, got their messages in logs and in see them Advanced Options, but still no indication of Last Attempted proof and nothing (zeroes) in `chia farm summary`.

**Reply by joehuchina** [reply]
have you found the solution ?

**Reply by salsaly4** [reply]
Aaaah, yes, it not a solution for now, but maintainers of the project are aware of this, this is not yet implemented and they plan an enhancement so this will work someday.

You can see the issue I've opened here — #2522

**Comment by jryski** [comment]
I have my main node running as a docker, and I have another node on a windows machine, I modified the address in the yaml, and copied the certs, but I have no way to tell if it's connected. I can't see it on the CLI. I tried the command grep ~/.chia/mainnet/log/debug.log -e harvester, but all I see there is a warning for an orphaned farm ( that I can't seem to clean up). The "Harvester network" section of the GUI in windows only shows self, and the CLI doesn't seem to have a command to check?
